Contexts/MainAppContext.cs

```diff
@@ -143,6 +143,28 @@ protected override void OnModelCreating(ModelBuilder builder)
                                                        .HasForeignKey(sn => sn.FreelancerId)
                                                        .HasPrincipalKey(u => u.Id)
                                                        .OnDelete(DeleteBehavior.NoAction);
+            /////
+            builder.Entity<TaskApprovalNotification>().HasOne<TaskEntity>()
```

Comment:
I suppose that the relationship between `TaskApprovalNotification` and `TaskEntity` should be one-to-one, but NVM, we can change it later if necessary